clusterloader2/pkg/measurement/common/simple/metrics_for_e2e.go

```diff
@@ -126,9 +131,7 @@ func (*metricsForE2EMeasurement) String() string {
 	return metricsForE2EName
 }
 
-type metricsForE2E metrics.MetricsCollection
-
-func (m *metricsForE2E) filterMetrics() {
+func filterMetrics(m *metrics.MetricsCollection) {
```

Comment:
This should be a method on MetricsCollection

Reply:
The problem is that MetricsCollection is vendor so I can't add method to it.
I can either changed it other repo, update vendor or copy the whole code here, however I'm not sure if it will be worth the effort.